.travis.yml

```diff
@@ -18,8 +18,8 @@ sudo: false
 # but since Travis only runs five jobs at a time throughout all of Exercism,
 # it is better citizenship to not test the additional versions.
 go:
-  - 1.7.5
-  - 1.8
+  - 1.8.3
+  - 1.9
```

Comment:
I think we have picked up some weirdness in the branch here. Perhaps a rebase to nextercism is in order? config especially? These travis changes shouldn't be here. Git can be tough sometimes!